.readthedocs.yaml

```diff
@@ -12,7 +12,7 @@ sphinx:
 
 # Set the version of python needed to build these docs.
 build:
-  os: "ubuntu-22.04"
+  os: "ubuntu-latest"
   tools:
     python: "3.8"
```

Comment:
not related to this PR but it's probably worth updating this to one of the newer python versions.